apps/bare-expo/app.json

```diff
@@ -47,6 +48,9 @@
     "runtimeVersion": {
       "policy": "appVersion"
     },
+    "experiments": {
+      "edgeToEdge": false
```

Comment:
It may be worth highlighting that this is an Android-specific feature (and maybe link to the official android docs for more
info)

Reply:
It's indeed very specific to Android, as edge to edge is the default on iOS (so for the moment, it has no effect on this platform). But as the goal of this feature is to unify how the system UI / keyboard is handled on both platforms, and as the `SystemBars` component will also be responsible for iOS config updates (bars styles), I think it's better to not scope this to android only.